cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -260,3 +289,37 @@ def test_custom_button_automate(request, appliance, context, submit, objects, bu
                 assert False, "Expected {count} requests not found in automation log".format(
                     count=str(expected_count)
                 )
+
+
+@pytest.mark.meta(blockers=[BZ(1659452, forced_streams=["5.9", "5.10"],unblock=lambda button_group: "group" not in button_group)])
```

Comment:
`button_group` is a tuple of objects, I don't think this unblock lambda is effective?

Reply:
yes, In such case this is best option. we are not upgraded `pytest`; with this version, we have to go with this. @mshriver I tested this by running locally. but still, if you have any suggestion please let me know.